root/components/forms.js

```diff
@@ -191,7 +188,7 @@ const FormRowSelectList = ({
   <div className="row">
     <label>{addColon(label)}</label>
     <div className="form-row-select-list">
-      <For each="subfield" index="index" of={repeatable.field.toArray()}>
+      {repeatable.field.toArray().map((subfield, index) => (
```

Comment:
I didn't factor out the `map` callback here, as done above, because it references other props in the closure.